inginious/common/tasks_problems.py

```diff
@@ -172,8 +207,28 @@ def __init__(self, task, problemid, content, translations=None):
         self._boxes = [
             self._create_box("", {"type": "file", "max_size": content.get("max_size", None), "allowed_exts": content.get("allowed_exts", None)})]
 
-    def get_type(self):
-        return "code-file"
+    def input_type(self):
+        return dict
+
+    @classmethod
+    def get_type(cls):
+        return "code_file"
```

Comment:
Let's replace it by "file", as it is not only used to transfer files containing human-readable code.